linera-execution/src/runtime.rs

```diff
@@ -536,11 +536,12 @@ where
         let value_future = tokio::task::spawn_blocking(move || {
             code.handle_query(query_context, runtime_sender, argument)
         });
-        runtime_actor.run().await?;
```

Comment:
We could add `// TODO(#989): simplify after message failures are not ignored` or equivalent in all those places

Reply:
Good point. Done :tada: